js/tests/visual/tab.html

```diff
@@ -159,47 +159,14 @@ <h4>Tabs with fade (no initially active pane)</h4>
         </div>
       </div>
 
-      <h4>Tabs with nav (with fade)</h4>
```

Comment:
Removed this as it's a weirdly invalid example, markup-wise. Not sure what this was supposed to show/visually test for...

Reply:
Maybe just nav without ul?

Reply:
right, fixed up (using the suggested pattern from the example using `<nav>` in https://getbootstrap.com/docs/5.0/components/navs-tabs/#javascript-behavior, and removing the dropdown) and reintroduced the visual test. this is now good to go from my end i'd say